src/main/java/it/gov/pagopa/pu/bff/mapper/TaxonomyCodeMapper.java

```diff
@@ -0,0 +1,28 @@
+package it.gov.pagopa.pu.bff.mapper;
+
+import it.gov.pagopa.pu.bff.dto.generated.TaxonomyCodeDTO;
+import it.gov.pagopa.pu.p4pa_organization.dto.generated.CollectionModelTaxonomyCodeDTOEmbedded;
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TaxonomyCodeMapper {
+
+  public List<TaxonomyCodeDTO> map(CollectionModelTaxonomyCodeDTOEmbedded input) {
```

Comment:
don't write a mapper to take a list and return a list, write a mapper for a single DTO (this will simplify also the test on the mapper)